src/desktop/components/split_test/running_tests.coffee

```diff
@@ -54,4 +54,14 @@ module.exports = {
     control_group: 'control'
     edge: 'experiment'
     weighting: 'equal'
+
+  search_bar:
+    key: 'search_bar'
```

Comment:
the experiment needs a new name (eg `search_bar_2`) otherwise the cookie assignment from our previous test as well as all the event data could get conflated.